terraform/modules/elasticsearch_broker/log_group.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_cloudwatch_log_group" "audit_log" {
+  name = "cloudwatch_arn"
```

Comment:
This name doesn't seem very specific? Shouldn't it be something like `elasticsearch_broker_audit_log` ?